commands.go

```diff
@@ -209,9 +210,39 @@ func (cmd *Command) restoreResolvConf(sum *[sha256.Size]byte) error {
 }
 
 func (cmd Command) Run(label string, cmdline ...string) error {
-	q := newQemuHelper(cmd)
-	q.Setup()
-	defer q.Cleanup()
+	var emulated bool
+	switch cmd.Architecture {
+	case "":
+		emulated = false
+	case "amd64":
+		emulated = runtime.GOARCH != "amd64"
+	case "armhf", "armel", "arm":
+		emulated = runtime.GOARCH != "arm" || runtime.GOARCH != "arm64"
```

Comment:
Not all armv8 systems can run in 32 bit mode (aarch32);  So assuming you don't have to run emulated isn't generally true here

Reply:
Do you know what systems can't? How can I detect that?

Reply:
Some of the serverclass ARM chips can't; Cavium ThunderX is the main example i know of.

For detection there are some notes on: https://www.spinics.net/lists/arm-kernel/msg746790.html